### PR TITLE
[release] v7.23.1

### PR DESCRIPTION
**Pull request by michelengelen**
### Prepare release

1. [x] Add changelog entry
2. [x] Clean changelog entry
3. [x] Update the root `package.json`'s version
4. [x] Update other `package.json`s ´version
5. [x] Open PR with changes and wait for review and green CI.
6. [ ] Announce a merge freeze.
7. [ ] Merge PR.

### Release the packages

1. [ ] Checkout the last version of the working branch
2. [ ] Build release packages (`pnpm i && pnpm release:build`
3. [ ] Publish release on npm (`pnpm release:publish`)
4. [ ] Push the newly created tag (`pnpm release:tag`)

### Publish docs and github release
1. [ ] Publish the documentation
2. [ ] Publish GitHub release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,72 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.23.1
+_Dec 5, 2024_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve German locale on the Data Grid component
+- 🐞 Bugfixes
```
Comment (michelengelen) [inline CHANGELOG.md:12]: Do we have any other highlights?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,72 @@
...
+Following are all team members who have contributed to this release:
+@arthurbalduini, @cherniavskii, @JCQuintas, @LukasTy and @MBilalShafi.
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.23.1`
+
+- [DataGrid] Improve SEO titles (#15695) @MBilalShafi
```
Comment (arminmeh) [inline CHANGELOG.md:26]: I think that this should be in [docs]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,73 @@
...
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve German locale on the Data Grid component
+- 🐞 Bugfixes
+
+Special thanks go out to the community contributors who have helped make this release possible:
+@lhilgert9.
+
+Following are all team members who have contributed to this release:
+@arthurbalduini, @cherniavskii, @JCQuintas, @LukasTy and @MBilalShafi.
```
[the remark this answers is not in the document]
Comment (michelengelen) [reply CHANGELOG.md:18]: 🫣 sry

[the remark this answers is not in the document]
Comment (flaviendelangle) [reply CHANGELOG.md]: :laughing: I just added a PR that I've done, I was not in the list before

File: CHANGELOG.md (hunk 1 above)
Comment (flaviendelangle) [reply CHANGELOG.md:12]: Not for eXplore
For a patch I think the current one are fine :ok_hand:

**Review by flaviendelangle — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,77 @@
...
+
+- [charts] Improve SVG `pattern` and `gradient` support (#15724) @JCQuintas
+
+#### `@mui/x-charts-pro@7.23.1` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-charts@7.23.1`.
+
+### Tree View
+
+#### `@mui/x-tree-view@7.23.1`
```
Comment (arminmeh) [inline CHANGELOG.md:61]: TreeView is not bumped, so this section should be removed

**Review by arminmeh — APPROVED** [review]

**Review by KenanYusuf — APPROVED** [review]

**Pull request merged**